sale_blanket_order_readonly_states/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,5 @@
+- Open Sales>Order>Blanket Orders
+- Create a new Blanket Order
+- With the module installed, changes can still be made to some fields until a Sales Order is issued
```

Comment:
Can you remove "With the module installed, changes can still be made to some fields until a Sales Order is issued"

This is a observation and not a test action.